roles/openshift_descheduler/tasks/upgrade.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Check if descheduler cronjob exists
+  oc_obj:
+    state: list
+    kind: cronjob
+    name: "{{ openshift_descheduler_cronjob_name }}"
+    namespace: "{{ openshift_descheduler_namespace }}"
+  register: descheduler_cronjob
+
+- name: Upgrade descheduler cronjob
+  import_playbook: install_descheduler.yaml
```

Comment:
Importing playbook inside a role?

Reply:
yes right, that's what i changed. thanks.